Sources/FTPropertyWrappers/InternetPassword.swift

```diff
@@ -3,25 +3,47 @@ import Foundation
 @propertyWrapper
 open class InternetPassword<T: Codable>: KeychainItemPropertyWrapper<T> {
 
-    @QueryElement(key: kSecAttrAccount) open var account: String?
```

Comment:
Does it even make sense to make all these properties mutable? If it in some cases doe not work properly?

Reply:
Those Query Elements are supposed to be read-only in the Swift code but do not have "read only" behaviour in keychain query such as creation date. QueryElement describes rules defined by keychain documentation. Read-only semantics in that case is, that if such element is inside a keychain query, such query is rejected with appropriate OSStatus (invalid request). This does not apply for kSecAttrAccount and others, which are not read only because we use them as a keys. As for declaration, to my knowledge, wrapped properties may not be marked as "let".

Reply:
This didn't make it much clearer to me. Mutability in this case can be achieved by not making the setter open, if desired.